pyproject.toml

```diff
@@ -45,6 +45,7 @@ dependencies = [
     "rich>=13.7.1",
     "numpy>=1.26.0,<2.2; python_version == '3.9'",
     "numpy>=1.26.0; python_version >= '3.10'",
+    "pytest-asyncio>=0.26.0",
```

Comment:
I can merge if this is urgent(?), but should this be a dev-only dependency? It doesn't seem important for users

Reply:
ah good catch! moved